### ISSUE TITLE
Streamfield menu hides plus icons when closed with no elements.

### ISSUE THREAD

**Issue by unexceptable** [heymonkeyriot/wagtailclearstream#11]
By default Wagtail if you close a streamfield menu when there are no elements, you get left behind a small single plus icon:
![image](https://user-images.githubusercontent.com/6404853/26957665-a0dc6996-4d1a-11e7-8b44-3dfd6d37a039.png)

Same in the case of a nested streamfield (although it looks awful):
![stream_menu](https://user-images.githubusercontent.com/6404853/26957706-e6dda766-4d1a-11e7-8d92-43f6da7f4cc6.png)

With the clearstream there is no icon:
![image](https://user-images.githubusercontent.com/6404853/26957745-1ef338dc-4d1b-11e7-9588-196a0453d5c4.png)

Same with the nested clearstream:
![no_icon](https://user-images.githubusercontent.com/6404853/26957784-4d26df42-4d1b-11e7-80f0-e7c86ad647e1.png)

**Comment by hminnovation** [comment]
Yes, it's an annoying bug. Thanks for flagging. Will look in to how I can improve.
